Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version: 5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Rudder-CustomerIO",
+    platforms: [
+        .iOS("13.0")
+    ],
+    products: [
+        .library(
+            name: "Rudder-CustomerIO",
+            targets: ["Rudder-CustomerIO"]),
+    ],
+    dependencies: [
+        .package(name: "Customer.io", url: "https://github.com/customerio/customerio-ios", .exact("3.2.2")),
```

Comment:
I can see in the CustomerIO documentation (refer [here](https://customer.io/docs/sdk/ios/quick-start-guide/#1-add-dependencies-in-xcode)) they mention to add the following link:
```
https://github.com/customerio/customerio-ios.git
```

Is the current implementation the same? (I think `.git` is missing.)